eng/scripts/Language-Settings.ps1

```diff
@@ -57,50 +57,6 @@ function Get-AllPackageInfoFromRepo ($serviceDirectory)
   return $allPackageProps
 }
 
-function Get-python-PackageInfoFromRepo  ($pkgPath, $serviceDirectory, $pkgName)
```

Comment:
Do we think it is worth keeping this helper for an individual project?

Reply:
Since we are now always calling `Get-AllPkgProperties` we wil never need to use this. Except we create a new method that works with it.

Reply:
Ok I'm find removing it as I agree I don't know of any real usages for it right now.